{
    "type": "bundle",
    "id": "bundle--3b11c997-3c7f-49e0-aec4-a8b1871a7473",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--5f1407d8-1bb2-4b76-9cd6-fe408bcb0154",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:12:07.288522Z",
            "modified": "2021-03-13T20:12:07.288522Z",
            "name": "CVE-2019-18981",
            "description": "Pimcore before 6.2.2 lacks an Access Denied outcome for a certain scenario of an incorrect recipient ID of a notification.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2019-18981"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}